install/linux/docker-ce/fedora.md

```diff
@@ -35,7 +35,12 @@ installed, uninstall them, along with associated dependencies.
 
 ```bash
 $ sudo dnf remove docker \
+                  docker-client \
+                  docker-client-latest \
                   docker-common \
+                  docker-latest \
+                  docker-latest-logrotate \
```

Comment:
I was in doubt for the logrotate package, but rotating logfiles outside of the daemon could be troublesome, and definitely not something we recommend. I'm open to removing this one though

ping @cpuguy83 wdyt?

Reply:
Works for me.